### PR TITLE
misc: Clean-up and minors pre 1025

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description
Some minor changes and updates pre 1030 runtime upgrade
Fixes #(issue)

## Changes and Descriptions
* Register new canonical Circle `USDC` on Celo
* Rename existing wormhole `USDC`
  * NOTE: This version is intentionally NOT a variant of `LocalUSDC`
* Bumb version of dev runtime
* Allow Council, root and LpAdmin to set `MarketFeeder` in the order book for now 

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: libs/types/src/tokens.rs [hunk 1]
```diff
@@ -452,7 +452,10 @@ pub mod usdc {
 	pub const CURRENCY_ID_LP_ETH_GOERLI: CurrencyId = CurrencyId::ForeignAsset(100_001);
 	pub const CURRENCY_ID_LP_BASE: CurrencyId = CurrencyId::ForeignAsset(100_002);
 	pub const CURRENCY_ID_LP_ARB: CurrencyId = CurrencyId::ForeignAsset(100_003);
-	pub const CURRENCY_ID_LP_CELO: CurrencyId = CurrencyId::ForeignAsset(100_004);
+
+	pub const CURRENCY_ID_LP_CELO_WORMHOLE: CurrencyId = CurrencyId::ForeignAsset(100_004);
+	pub const CURRENCY_ID_LP_CELO: CurrencyId = CurrencyId::ForeignAsset(100_006);
```
Comment (lemunozm) [inline libs/types/src/tokens.rs:457]: `CURRENCYT_ID_LP_CELO` was `100_004`, and now it is `100_006`. I guess this is the intention of the migration, so wherever we used `CURRENCY_ID_LP_CELO`, now it's `CURRENCY_ID_LP_CELO_WORKHOLE`.

**Review by lemunozm — APPROVED** [review]
LGTM!

**Review by wischli — APPROVED** [review]
LGTM!

File: libs/types/src/tokens.rs (hunk 1 above)
Comment (mustermeiszer) [reply libs/types/src/tokens.rs:457]: Exactly!

**Review by wischli — APPROVED** [review]

**Pull request merged**